ebpf/session.go

```diff
@@ -167,16 +167,19 @@ func (s *session) Start() error {
 	}
 
 	_, nsIno, err := getPIDNamespace()
-	if err != nil {
-		return fmt.Errorf("unable to get pid namespace %w", err)
-	}
-	err = spec.RewriteConstants(map[string]interface{}{
-		"global_config": pyrobpf.ProfileGlobalConfigT{
-			NsPidIno: nsIno,
-		},
-	})
-	if err != nil {
-		return fmt.Errorf("pyrobpf rewrite constants %w", err)
+	// if the file does not exist, CONFIG_PID_NS is not supported, so we just ignore the error
+	if os.IsExist(err) {
```

Comment:
Something looks off here. I would expect this condition to be true if error is ErrExist and I do not expect this error to be returned during pid ns check ( we do not create any files/objects). 
The next error check looks suspicious as well. It looks like we rewrite global_config constant here only in case if ErrExist and err==nil which looks impossible to my brain 
Did you test this change with PD_NS configured and inside a pid namespace ( a docker container for example)?

We need to change something here.
Let me know if I am missing/misunderstanding something.

Reply:
PR Updated. I'm new to golang and it's my misunderstanding of `os.IsExist`. It's now replaced by `!os.IsNotExist()`.

Reply:
I tested the new patch on WSL and Android Phone, both work fine.